library/src/main/jni/libpng/arm/filter_neon.S

```diff
@@ -58,7 +57,6 @@ ELF     .size   \name, . - \name
         .global \name
     .endif
 ELF     .type   \name, STT_FUNC
-        .func   \name
```

Comment:
Why delete this line?

Reply:
I could not compile the files with `.func` and `.funcend` (clang returned an error). See https://codereview.chromium.org/481243002

Reply:
OK, I get it.